lib/src/framework.dart

```diff
@@ -71,7 +71,6 @@ bool debugHotReloadHooksEnabled = true;
 ///
 ///   @override
 ///   void initState() {
-///     super.initState();
```

Comment:
That one and the `super.dispose` should stay

Reply:
oops my fault!